HoloRepositoryDemoApplication/Assets/Scripts/PatientListItem.cs

```diff
@@ -9,34 +9,43 @@ public class PatientListItem : MonoBehaviour
     [SerializeField]
     private TextMeshProUGUI PatientInfo = null;
     [SerializeField]
-    private Text PatientID = null;
+    private RawImage Avatar = null;
     [SerializeField]
-    private TextMeshProUGUI PatientName = null;
+    private Texture2D Boy = null;
+    [SerializeField]
+    private Texture2D Girl = null;
+    private Patient Patient;
+
     public static List<Hologram> HologramsList = new List<Hologram>();
 
     private void Start()
     {
         PatientInfo = GetComponent<TextMeshProUGUI>();
     }
 
-    public void SetName(string name)
-    {
-        PatientName.text = name;
-    }
-
     public void SetText(string Patientinformation)
     {
         PatientInfo.text = Patientinformation;
     }
-    public void SetID(string id)
+    public void SetPatient(Patient patient)
+    {
+        Patient = patient;
+    }
+
+    public void SetImage(string gender)
     {
-        PatientID.text = id;
+        if (gender == "male")
```

Comment:
This is not the most important issue so don't necessarily do it now, just put it to your personal backlog if you like: I think you might want to have a default pic, for cases other than `"male"` or `"female"`. Also, later on we should change the Accessor API to include the imageUrl in the patient response.